### ISSUE TITLE
Deleting projects returns HTTP 500 and fails to delete quotas

### ISSUE THREAD

**Issue by xtreme-conor-nosal** [goharbor/harbor#12292]
**Expected behavior and actual behavior:**
Expected: Deleting a project should complete successfully
Actual: Delete returns HTTP 500

**Steps to reproduce the problem:**
Create a project
Delete the project

**Versions:**
Please specify the versions of following systems.

- harbor version: [1.10.3]
- harbor helm: [1.3.3]
- GKE

**Additional context:**

Logs indicate:
```
2020-06-19T16:16:00Z [WARNING] [/common/quota/manager.go:278]: Load quota reference object (project, 83) failed: project not found, project_id: 83
2020-06-19T16:16:00Z [ERROR] [/common/api/base.go:233]: failed to get quota manager: project not found, project_id: 83
2020-06-19T16:16:00Z [ERROR] [/common/api/base.go:73]: DELETE /api/projects/83 failed with error: {"code":500,"message":"Internal Server Error"}
2020-06-19 12:16:00.030 EDT remote_addr - "DELETE /api/projects/83 HTTP/1.1" 500 47 "-" "-" 0.102 0.102 .
```

It looks like the project model is deleted, and then the associated quota manager is loaded:
https://github.com/goharbor/harbor/blob/v1.10.3/src/core/api/project.go#L295-L300
but the quota's loader driver tries to load the project model again:
https://github.com/goharbor/harbor/blob/v1.10.3/src/common/quota/driver/project/driver.go#L53

**Comment by guillaumelfv** [comment]
Hi, got the same issue with harbor 1.10.1 !

**Comment by reasonerjt** [comment]
@heww do we want this fix in 2.0.2?

**Comment by heww** [comment]
This issue only exists in v1.10.x and v1.9.x.
